source/write-to-mongodb.txt

```diff
@@ -28,6 +28,18 @@ Write to MongoDB
 
          .. include:: /scala/write-to-mongodb.txt
 
+.. warning::
+
+   If you specify the ``overwrite`` save mode, the connector drops the target
+   collection and creates a new collection that uses the
+   default collection options.
```

Comment:
what are the default collection options?

Reply:
I'm not sure, but I asked in the drivers ticket and can include if you think it's important for this page.

Reply:
If there's a Server link that describes what a default collection is, it could be useful to link, but not necessary. I think this [page](https://www.mongodb.com/docs/manual/reference/method/db.createCollection/) has some info